src/main/java/org/brapi/test/BrAPITestServer/service/pheno/ScaleService.java

```diff
@@ -39,16 +39,17 @@ public ScaleService(ScaleRepository scaleRepository, OntologyService ontologySer
 
 	public List<Scale> findScales(String scaleDbId, String observationVariableDbId, String ontologyDbId,
 			String commonCropName, String programDbId, String externalReferenceId, String externalReferenceID,
-			String externalReferenceSource, Metadata metadata) {
+			String externalReferenceSource, Metadata metadata)
+		throws BrAPIServerException {
 		Pageable pageReq = PagingUtility.getPageRequest(metadata);
 		SearchQueryBuilder<ScaleEntity> searchQuery = new SearchQueryBuilder<ScaleEntity>(ScaleEntity.class);
 		if (observationVariableDbId != null) {
-			searchQuery = searchQuery.join("variables", "variable").appendSingle(observationVariableDbId,
+			searchQuery = searchQuery.join("variables", "variabTle").appendSingle(observationVariableDbId,
```

Comment:
This looks like it was a simple typo.
```suggestion
			searchQuery = searchQuery.join("variables", "variable").appendSingle(observationVariableDbId,
```

Reply:
Good catch 😳